metadata/src/main/java/org/apache/kafka/controller/ConfigurationControlManager.java

```diff
@@ -434,6 +434,14 @@ Map<String, String> getConfigs(ConfigResource configResource) {
         }
     }
 
+    String getTopicConfig(String topicName, String configKey) {
```

Comment:
This is not correct. If you want to know what the topic configuration is, you have to use `getEffectiveTopicConfigValue`. This will handle things like the N levels of config overrides.

I also don't agree with mapping "doesn't exist" to "empty string" (we don't do that anywhere else in the code)

Finally, you'll need a strategy for handling changes to these dynamic configurations

Reply:
- Now using the computeEffectiveTopicConfigs. getEffectiveTopicConfigValue is not in AK yet.
- return null if the config is not found.
- It will be a separate change to handle the ELR change if minISR is dynamically updated.

Reply:
ok